### DISCUSSION TITLE
Share my timezone

### DISCUSSION THREAD

**Discussion by CutSnake01** [JulianPrieber/littlelink-custom#802] [Feature requests]
I'm using linkstack more or less as a business card. It's got all my social links and also contact info such as phone number and email. It would be great to be able to show people what my current time zone is, so that if they're trying to contact me they have some idea of our likely I am to answer or respond.

To take it a step further, it'd be great to show two timezones: 'My Home Timezone' (where I live normally) and 'My Current Timezone' (the time where ever I am in the world at that moment for when I'm travelling.)

Some neat little analogue clock style graphic at the bottom of screen would look great. Or if there were a few different skin options for different clock styles to choose from that'd be awesome.

**Comment by JulianPrieber** [comment]
Not a bad idea.
![chrome_EWEi623IUE](https://github.com/user-attachments/assets/0a0c875f-aa7b-4039-b3f9-ee3b48ecc294)
I could imagine a display like the timezone here on your GitHub profile.
